src/System.Runtime.Intrinsics/ref/Configurations.props

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PackageConfigurations>
```

Comment:
Remove all of the PackageConfigurations, and only leave the BuildConfigurations. Actually I would start having just:
```xml
  <PropertyGroup>
    <BuildConfigurations>
      netcoreapp;
    </BuildConfigurations>
  </PropertyGroup>
```